components/messaging/ConversationWrapper.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+import { useContext } from 'react';
+import MeatballIcon from '../icons/MeatballIcon';
+import PlusIcon from '../icons/PlusIcon';
+import ConversationsList from './ConversationsList';
+import CurrentConversation from './CurrentConversation';
+import useConversation from '@/app/(dashboard)/conversations/useConversation';
+import useMediaQuery from '../hooks/useMediaQuery';
+
+const ConversationWrapper: React.FC = () => {
+  const isBreakpoint = useMediaQuery(1000);
+  const { showConversationList, setShowConversationList } =
+    useContext(useConversation);
+  return (
+    <>
+      <div className='mt-4 flex justify-between px-3 '>
+        <button>
+          <PlusIcon width={45} height={45} />
+        </button>
+        <button>
+          <MeatballIcon width={35} height={35} />
+        </button>
+      </div>
+      {!isBreakpoint && (
+        <div className='mt-4'>
+          <ConversationsList />
+          <>
+            <CurrentConversation />
+          </>
+        </div>
+      )}
+      {isBreakpoint && (
+        <div className='mt-4'>
+          {showConversationList === true ? (
```

Comment:
Petty request to homogenise our names and make this `showConversationsList` in the plural to match the component it's referring to.

Actually, probably the same for `ConversationsWrapper`

Reply:
Agree for the showConversationsList (as ConversationsList is plural) but not sure it makes sense for the wrapper as neither ConversationProvider nor ConversationCard is plural

Reply:
That's a good enough compromise for me